apps/patient/src/views/Pharmacy.tsx

```diff
@@ -99,90 +101,124 @@ export const Pharmacy = () => {
     lat: number | undefined
     lng: number | undefined
   }) => {
+    // Reset view if search location changes
     if (loc && loc !== location) {
-      reset()
+      if (lat && lng) {
```

Comment:
maybe `if (loc && loc !== location && lat && lng) {`

Reply:
gonna stick with this for now, it's a bit more readable for me